### PR TITLE
internal: document review requesting etiquette

### PR DESCRIPTION
**Pull request by matklad**
* don't feel obliged to quickly review every PR assigned to you
* so that other folks can notify you about interesting PRs without
  thinking to much about creating additional work for you

### CODE REVIEW

**Comment by matklad** [comment]
WDYT?

The problem I want to solve is that we have a sizable queue of PRs, and it's unclear who is default reviewer for them. Ie, when I have my slot for reviews, it's unclear to me which PRs already have someone intending to r+, and which PRs are blocked on no one feeling confident enough to review them. 

I wish I could just sort by PRs assigned to me, and then triage the PRs without updates.

File: docs/dev/README.md
Comment (lnicola) [inline docs/dev/README.md:245]:
```suggestion
  Feel free to request a review or assign any PR to a reviewer with the relevant expertise to bring the work to their attention.
```

Other contributors won't be able to assign reviewers, but some of them already have an idea about who should take a look at the code.

**Review by lnicola — APPROVED** [review]
I don't know if this will speed up the reviews, but I don't see why it might be objectionable.

**Comment by flodiebold** [comment]
This might be a good place to mention not pinging people in PR messages (e.g. with "r? @flodiebold") since that leads to spam whenever someone pulls the PR merge commit into their fork? :grimacing:

**Pull request merged**
